### PR TITLE
Update to Telegram json file to change text in labels and descriptions.

### PR DESCRIPTION
**Pull request by TovaCohen**
I made some updates to the json file for the Telegram connector. One update corrects a typo. The other updates hopefully clarify the text and apply consistent style formats. 
Ready to merge.

### CODE REVIEW

File: app/connector/telegram/src/main/resources/META-INF/syndesis/connector/telegram.json [hunk 1]
```diff
@@ -72,15 +72,15 @@
       "type": "MAVEN"
     }
   ],
-  "description": "Send and receive messages to/from a Telegram chat",
+  "description": "Receive and send messageses.",
```
Comment (oscerd) [inline app/connector/telegram/src/main/resources/META-INF/syndesis/connector/telegram.json]: I think this is wrong, otherwise this looks good.

Comment (TovaCohen) [reply app/connector/telegram/src/main/resources/META-INF/syndesis/connector/telegram.json]: Glad you caught that!

**Comment by oscerd** [comment]
Yes, sure. I think it should be fine. Not sure if the release for 1.5.x has been already built. @heiko-braun @paoloantinori ?

**Comment by paoloantinori** [comment]
we are not planning to add other commits to 1.5.x release, but since it often happens that exceptional situations pop up at the last second, what we can do is to keep track of commits we can consider do include, if the opportunity happens.

that's the best we can do for this specific request.

**Comment by heiko-braun** [comment]
I think there will be another CR, but I need @janstey to confirm this. However it doesn't justify the inclusion since `1.5.x` is in blockers only mode. 

I see this fix is simple, unlikely with side-effects, but I would still prefer to stick to the "rules". Before the release is after the release. The next window comes quickly and we clan include it there.

**Pull request merged**